pkg/controller/ps/tls.go

```diff
@@ -25,7 +25,17 @@ import (
 func (r *PerconaServerMySQLReconciler) ensureTLSSecret(ctx context.Context, cr *apiv1alpha1.PerconaServerMySQL) error {
 	log := logf.FromContext(ctx)
 
-	err := r.ensureSSLByCertManager(ctx, cr)
+	secret := &corev1.Secret{}
+	err := r.Client.Get(ctx, types.NamespacedName{
+		Namespace: cr.Namespace,
+		Name:      cr.Spec.SSLSecretName,
+	}, secret)
+
+	if cr.Spec.SSLSecretName != "" && err == nil {
```

Comment:
this way we won't ever update the secret, no matter if it's created by user or operator

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/blob/main/pkg/controller/pxc/tls.go#L48-L51